clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gitRepo/GitJobExecutor.java

```diff
@@ -110,6 +134,34 @@ public void clone(String repoUrl, String branch, Path destination) throws IOExce
     }
   }
 
+  public void pull(String repoUrl, Path localPath) throws IOException {
```

Comment:
This could be `private` (just like `clone` above - although it may be too late for that) unless you think it's useful to expose.